README.md

```diff
@@ -57,45 +56,44 @@ Other features include:
 - **Powerful API & Webhooks**: Extend the capability of the software using Chatwoot’s webhooks and APIs.
 - **Integrations**: Chatwoot natively integrates with Slack right now. Manage your conversations in Slack without logging into the dashboard.
 
----
+## Documentation
 
-### Documentation
+Detailed documentation is available at [chatwoot.com/help-center](https://www.chatwoot.com/help-center).
 
-Detailed documentation is available at [www.chatwoot.com/help-center](https://www.chatwoot.com/help-center).
-
-### Translation process
+## Translation process
 
 The translation process for Chatwoot web and mobile app is managed at [https://translate.chatwoot.com](https://translate.chatwoot.com) using Crowdin. Please read the [translation guide](https://www.chatwoot.com/docs/contributing/translating-chatwoot-to-your-language) for contributing to Chatwoot.
 
----
-
-### Branching model
+## Branching model
 
 We use the [git-flow](https://nvie.com/posts/a-successful-git-branching-model/) branching model. The base branch is `develop`.
 If you are looking for a stable version, please use the `master` or tags labelled as `v1.x.x`.
 
----
+## Deployment
 
-### Deployment
-
-#### Heroku one-click deploy
+### Heroku one-click deploy
 
 Deploying Chatwoot to Heroku is a breeze. It's as simple as clicking this button:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/chatwoot/chatwoot/tree/master)
 
 Follow this [link](https://www.chatwoot.com/docs/environment-variables) to understand setting the correct environment variables for the app to work with all the features. There might be breakages if you do not set the relevant environment variables.
 
-#### Other deployment options
+### Other deployment options
 
 Please follow [deployment architecture guide](https://www.chatwoot.com/docs/deployment/architecture) to deploy with Docker or Caprover.
 
----
-#### Security
+## Security
+
 Looking to report a vulnerability? Please refer our [SECURITY.md](./SECURITY.md) file.
 
----
-### Contributors ✨
+
+## Community? Questions? Support ?
+
+If you need help or just want to hang out, come say hi on our [ discord.](https://discord.gg/cJXdrwS)
```

Comment:
```suggestion
If you need help or just want to hang out, come, say hi on our [Discord](https://discord.gg/cJXdrwS) channel.
```

Reply:
Channels do not sound right 😅  Discord server ?

Reply:
'server' / 'channels' sounds good.